miden-lib/src/lib.rs

```diff
@@ -5,7 +5,10 @@ extern crate alloc;
 
 use assembly::{utils::Deserializable, Library, LibraryNamespace, MaslLibrary, Version};
 
+pub mod assembler;
```

Comment:
It seems like this `assembler` module is used only for tests/mocks. If so, we should probably put it behind `testing` feature (or move into the `testing` module below).

Reply:
This will likely be necessary by the client and the rollup. I think we should just leave it in, and use the same assembler code for those components.

Reply:
Good point. Let's keep it.